service/tbc/rpc.go

```diff
@@ -259,18 +294,21 @@ func (s *Server) handleUtxosByAddressRequest(ctx context.Context, req *tbcapi.Ut
 			e := protocol.NewInternalError(err)
 			return &tbcapi.UtxosByAddressResponse{
 				Error: e.ProtocolError(),
-			}, e
+			}, err
```

Comment:
The internal error (`e`) should be returned as the error here, as it contains the unique trace for the error.

Reply:
ah good catch, thanks